public/tabs/history/chat_history_page.tsx

```diff
@@ -111,26 +111,17 @@ export const ChatHistoryPage: React.FC<ChatHistoryPageProps> = React.memo((props
     };
   }, [props.shouldRefresh, services.conversations]);
 
+  useEffect(() => {
```

Comment:
Shall we change useUpdateEffect here? I think the dataSourceIdUpdates will have a default value. The state change will be fired with component did mount.